bigquery_etl/parse_udf.py

```diff
@@ -34,32 +34,55 @@ class RawUdf:
     def from_file(filepath):
         """Read in a RawUdf from a SQL file on disk."""
         dirpath, basename = os.path.split(filepath)
+
+        with open(filepath) as f:
+            text = f.read()
+
+        sql = sqlparse.format(text, strip_comments=True)
+        statements = [s for s in sqlparse.split(sql) if s.strip()]
```

Comment:
`if s.strip()` doesn't do anything here
```suggestion
        statements = [s for s in sqlparse.split(sql)]
```